client/src/App.js

```diff
@@ -10,6 +11,7 @@ const App = () => (
       <NavBar />
       <Switch>
         <Route exact path="/" component={LandingPage} />
```

Comment:
The landing page is an empty component, no need to import it and write the path for it
You can delete for now, and the one who creates the landing page will import and use it